packages/apollo/src/commands/service/__tests__/check.test.ts

```diff
@@ -1,22 +1,328 @@
-import {
+import ServiceCheck, {
   formatHumanReadable,
   formatMarkdown,
   formatTimePeriod
 } from "../check";
 import checkSchemaResult from "../../../../__fixtures__/check-schema-result";
 import { ChangeSeverity } from "apollo-language-server/lib/graphqlTypes";
 import chalk from "chalk";
+import nock = require("nock");
+import { stdout, stderr } from "stdout-stderr";
+import * as graphql from "graphql";
+
+/**
+ * Single URL for all local requests to be mocked
+ */
+const localURL = "http://localhost:4000";
+
+/**
+ * Default API key. This is not an actual API key but a randomly generated string.
+ *
+ * If you need to use the `nock` recorder, then this will not work because we won't be able to access engine
+ * with a fake API key.
+ */
+const fakeApiKey = "service:engine:9YC5AooMa2yO11eFlZat11";
+
+/**
+ * Engine API key we're using.
+ *
+ * We should use the `fakeApiKey` whenever possible to ensure we can't accidentally access real APIs.
+ */
+const apiKey = process.env.ENGINE_API_KEY || fakeApiKey;
```

Comment:
Totally right and totally intentional. If we want to refresh the mock data we'll have to run this against the real APIs using a real `ENGINE_API_KEY`.

Reply:
Right, but if I'm running tests locally would I normally want to use `fakeApiKey` (per the comment)? I think if I ran this locally, _my_ api key would get loaded up just because I have the tooling repo configured for engine.

I think you're saying that's intentional, and if so then 👍. I was basing my comment on your comment about using `fakeApiKey` whenever possible.

Reply:
I'm not sure how we can refresh the mock data from this comment. Can we add something here and possibly the readme to explain how that process might work?

Reply:
Great suggestion :) And you're right Trevor; I'll remove the real API key and then document when to change that.